PROJECT_PROGRESSION.md

```diff
@@ -252,7 +224,18 @@ It is expected that incubating project will make an active effort to work throug
 
 1. consensus within private mailing list to move into incubation process
 
-## IV. Annual Review Process
+## IV. Growth Plan
+
+At Large project that are interested in reaching Impact Stage need to develop a growth plan to so do. This plan needs to be developed in collaboration with the CPC and approved by it.
```

Comment:
I think in the discussions, the idea was that a Growth plan is not tied to wanting to reach Impact Stage, and that even Impact projects might want a Growth plan.

Maybe

Project that are interested in extra support/attention in order to grow in a certain area need to developer a plan to do so....

Reply:
Agreed that what we discussed.

I'm curious however, what would the incentives be for a project to go through the foundation's structure for this? Why not just create their own roadmap?

Projects that are doing this to move to Impact stage are incentivized by the status/recognition aspect and additional CPC weight.

I think I'm inclined to suggest starting off with this a just a growth plan for At Large to Impact at first (as an MVP, if you will) and opening it up if there's demand.

Reply:
I think I agree with @mhdawson here. My understanding is that any project, even an Impact project, could adopt a growth plan. It could be that the growth plan is for an At-Large project to become an Impact project, but it also could just be to grow contributors, to reach a milestone, or a variety of any other reasons. Perhaps even moving to Emeritus could be considered a "growth plan". ¯\_(ツ)_/¯

Reply:
Also agree with @mhdawson. As I see it, a Growth Plan is a tool for which we presume that the most use will be for projects wanting to transition from At-Large to Impact, but there should not be a built-in limitation preventing that tool from being used for any other purpose, which we might not be able to imagine or exhaustively enumerate at this time.

Reply:
I like the idea of having a plan for each shift:

* Incubating -> At Large/Impact
* At Large -> Impact
* At Large/Impact -> Emeritus

They're different plans, though. I'm also less sure about the relevance of having a generic growth plan. Has anyone asked for this? Why don't we get there once someone actually asks?